src/jarabe/frame/clipboardicon.py

```diff
@@ -75,6 +76,8 @@ def get_object_id(self):
 
     def _drag_data_get_cb(self, widget, context, selection, target_type,
                           event_time):
+        frame = jarabe.frame.get_view()
+        self._timeout_id = GObject.timeout_add(jarabe.frame.frame._NOTIFICATION_DURATION, lambda: frame.remove_notification(self._notif_icon))
```

Comment:
1.  Lines should be max 80chars
2.  The _ in a variable signals it is private.  Maybe we need to change _NOTICICAION_DURATION to just NOTIFICATION_DURATION?